lib/fs.js

```diff
@@ -115,8 +115,8 @@ function rethrow() {
     var backtrace = new Error();
     return function(err) {
       if (err) {
-        backtrace.stack = err.name + ': ' + err.message +
-                          backtrace.stack.substr(backtrace.name.length);
+        backtrace.stack = `${err.name}: ${err.message}` +
+                        `${backtrace.stack.substr(`${backtrace.name.length}`)}`;
```

Comment:
It looks like this second line could be left as it was. Just the first line needs to be a template literal.

Reply:
Thanks for the review comments @apapirovski . Have made the necessary changes and did a fresh commit.